docs/source/user-guide/sql/scalar_functions.md

```diff
@@ -569,6 +569,7 @@ trunc(numeric_expression[, decimal_places])
 
 - [coalesce](#coalesce)
 - [nullif](#nullif)
+- [nvl/ifnull](#nvl/ifnull)
```

Comment:
This link to `nvl/ifnull` seems broken.
https://github.com/apache/arrow-datafusion/blob/f6c6e385041974c4e501bbe9e3ab42284ddddfda/docs/source/user-guide/sql/scalar_functions.md#conditional-functions

`[nvl/ifnull](#nvlifnull)` should work.

Reply:
not support `/`, I separated them， thank you again 😄